openjdk/openjdk.mk

```diff
@@ -156,3 +156,8 @@ endif
 ifeq (,$(wildcard $(PROBLEM_LIST_FILE)))
 	PROBLEM_LIST_FILE:=$(PROBLEM_LIST_DEFAULT)
 endif
+
+GRAAL_PROBLEM_LIST_FILE:=
```

Comment:
Add a comment here explaining why?

Reply:
https://github.com/adoptium/aqa-tests/issues/2845 describes the problem this PR fixes, this file only exists in jdk11 and jdk16, so when it is present, we will include it so that the tests it contains will be excluded.  When it does not exist, we won't attempt to pass it in (and hit an error that the file does not exist).

Are you asking @sophia-guo to add a comment in code, something like "Graal problem list only exists in jdk11 and jdk16" ?

Reply:
@karianna comment added to make it readable.